docs/csharp/tutorials/nullable-reference-types.md

```diff
@@ -34,7 +34,7 @@ The code you'll write for this sample expresses that intent, and the compiler en
 Create a new console application either in Visual Studio or from the command line using `dotnet new console`. Name the application `NullableIntroduction`. Once you've created the application, you'll need to specify that the entire project compiles in an `enabled` **nullable annotation context**. Open the `csproj` file and add a `Nullable` element to the `PropertyGroup` element. Set its value to `enabled`. You must opt into the **nullable reference types** feature, even in C# 8 projects. That's because once the feature is turned on, existing reference variable declarations become **non-nullable reference types**. While that decision will help find issues where existing code may not have proper null-checks, it may not accurately reflect your original design intent:
```

Comment:
``
```suggestion
Create a new console application either in Visual Studio or from the command line using `dotnet new console`. Name the application `NullableIntroduction`. Once you've created the application, you'll need to specify that the entire project compiles in an enabled **nullable annotation context**. Open the `csproj` file and add a `Nullable` element to the `PropertyGroup` element. Set its value to `enable`. You must opt into the **nullable reference types** feature, even in C# 8 projects. That's because once the feature is turned on, existing reference variable declarations become **non-nullable reference types**. While that decision will help find issues where existing code may not have proper null-checks, it may not accurately reflect your original design intent:
```

Reply:
The correct value for the `Nullable` element is `enable`.  This suggestion should be accepted.

Then, this is ready to merge.